_component_design/accessibility.md

```diff
@@ -0,0 +1,219 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Use headings and spacing to group related content
+
+When it comes to designing with space, remember that space as an aesthetic quality takes a backseat to space as a design tool. Space creates relationships, defines hierarchies, and emphasizes content. Always keep these in mind when considering how to reduce visual noise and create visual flow in your design. Some people with cognitive disabilities find it difficult to read text when the lines are close together. Providing extra space between lines and paragraphs allows them to better track the next line and to recognize when they have reached the end of a paragraph. WCAG prescribes that line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Helix minimum spacing standards require: 
+
+- Line spacing for Roboto Regular 16px is 24px.
+- Paragraph spacing is at least 36px. 
+- Use whitespace and proximity to make relationships between content more apparent.
+- Style headings to group content, reduce clutter, and make it easier to scan and understand.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "Little spacing and unclear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "More spacing and clear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Provide easily identifiable feedback
+
+Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Instructions should be easy to identify and suggestions to help fix issues whenever errors are returned must be provided. Important feedback that requires a user action should be presented in a prominent style.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Failed text inputs
+
+There are many acceptable ways to make text inputs visually accessible. If you’re indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the ‘error message’, the border color turns red and doubles in thickness from 1px to 2px for better accessibility.
```

Comment:
remove quotes around 'error message'

Reply:
Also, what is 'better accessibility' ? For clarity?

Reply:
Removed quotes

Reply:
Reword - "...from 1px to 2px to provide distinguishable visual queues not reliant on color"